Vic2ToHoI4/Data_Files/blankMod/output/common/autonomous_states/sphereling.txt

```diff
@@ -0,0 +1,44 @@
+autonomy_state = {
+	id = autonomy_sphereling
+	
+	min_freedom_level = 0.90
+
+	rule = {
+		can_not_declare_war = no
+		can_decline_call_to_war = yes
+	}
+	
+	modifier = {
+		autonomy_manpower_share = 0.0
+		extra_trade_to_overlord_factor = 0.25
+		overlord_trade_cost_factor = -0.25
+	}
+	
+	ai_subject_wants_higher = {
+		factor = 1.0
+	}
+	
+	ai_overlord_wants_lower = {
+		factor = 0.0
+	}
+
+	ai_overlord_wants_garrison = {
+		always = no
+	}
+
+	allowed = {
+		has_dlc = "Together for Victory"
+	}
+
+	allowed_levels_filter = {
+		autonomy_sphereling
+	}
+
+	can_take_level = {
+		always = yes
+	}
+
+	can_lose_level = {
```

Comment:
Will this prevent a country from breaking free from a sphere? Is that desired?

Reply:
The idea is to disallow falling into dominion and allow breaking from sphere. I'm currently working on adding the autonomy level based on vic2 influences.

Reply:
Ah, I read it backwards.